support/scripts/mkefi.py

```diff
@@ -2,183 +2,208 @@
-    "AddressOfEntryPoint"        : [0x0000000000000000, 0x04],  # Offset 104
-    "BaseOfCode"                 : [0x0000000000000000, 0x04],  # Offset 108
-
+    "Magic": (0x000000000000020B, 0x02),  # Offset  88
+    "MajorLinkerVersion": (0x0000000000000000, 0x01),  # Offset  90
+    "MinorLinkerVersion": (0x0000000000000000, 0x01),  # Offset  91
+    "SizeOfCode": [0x0000000000000000, 0x04],  # Offset  92
+    "SizeOfInitializedData": [0x0000000000000000, 0x04],  # Offset  96
+    "SizeOfUninitializedData": [0x0000000000000000, 0x04],  # Offset 100
+    "AddressOfEntryPoint": [0x0000000000000000, 0x04],  # Offset 104
+    "BaseOfCode": [0x0000000000000000, 0x04],  # Offset 108
     # Optional Header Windows-Specific Fields (Image Only)
-    "ImageBase"                  : [0x0000000000000000, 0x08],  # Offset 112
-    "SectionAlignment"           : (0x0000000000001000, 0x04),  # Offset 120
-    "FileAlignment"              : (0x0000000000001000, 0x04),  # Offset 124
+    "ImageBase": [0x0000000000000000, 0x08],  # Offset 112
+    "SectionAlignment": (0x0000000000001000, 0x04),  # Offset 120
+    "FileAlignment": (0x0000000000001000, 0x04),  # Offset 124
     "MajorOperatingSystemVersion": (0x0000000000000000, 0x02),  # Offset 128
     "MinorOperatingSystemVersion": (0x0000000000000000, 0x02),  # Offset 130
-    "MajorImageVersion"          : (0x0000000000000000, 0x02),  # Offset 132
-    "MinorImageVersion"          : (0x0000000000000000, 0x02),  # Offset 134
-    "MajorSubsystemVersion"      : (0x0000000000000000, 0x02),  # Offset 136
-    "MinorSubsystemVersion"      : (0x0000000000000000, 0x02),  # Offset 138
-    "Win32VersionValue"          : (0x0000000000000000, 0x04),  # Offset 140
-    "SizeOfImage"                : [0x0000000000000000, 0x04],  # Offset 144
-    "SizeOfHeaders"              : [0x0000000000000000, 0x04],  # Offset 148
-    "CheckSum"                   : (0x0000000000000000, 0x04),  # Offset 152
-    "Subsystem"                  : (0x000000000000000a, 0x02),  # Offset 156
-    "DllCharacteristics"         : (0x0000000000000000, 0x02),  # Offset 158
-    "SizeOfStackReserve"         : (0x0000000000000000, 0x08),  # Offset 160
-    "SizeOfStackCommit"          : (0x0000000000000000, 0x08),  # Offset 168
-    "SizeOfHeapReserve"          : (0x0000000000000000, 0x08),  # Offset 176
-    "SizeOfHeapCommit"           : (0x0000000000000000, 0x08),  # Offset 184
-    "LoaderFlags"                : (0x0000000000000000, 0x04),  # Offset 192
-    "NumberOfRvaAndSizes"        : (0x0000000000000000, 0x04),  # Offset 196
-
+    "MajorImageVersion": (0x0000000000000000, 0x02),  # Offset 132
+    "MinorImageVersion": (0x0000000000000000, 0x02),  # Offset 134
+    "MajorSubsystemVersion": (0x0000000000000000, 0x02),  # Offset 136
+    "MinorSubsystemVersion": (0x0000000000000000, 0x02),  # Offset 138
+    "Win32VersionValue": (0x0000000000000000, 0x04),  # Offset 140
```

Comment:
I have to say that sometimes the tool is too brutal with alignment. I find the previous block more readable.

Reply:
Robots show no compromise, unless you specify to them. If you can, look for any way to ignore lines, but not sure if there's something for this.

Me I see it as this: If as simple as possible (no formatting), everyone can style however they like when reading.
For example I did not agree with the spaces before the `:`, they should have been after it

Reply:
I would say take the hit an leave like this and look to add line ignores only if someone complains about readability.

Reply:
Actually, I think these robots are there to support us but not to replace. There are situations (like this alignment) where a human should be able to overwrite robot behavior. We do this with checkpatch on the other unikraft sources: in some cases we accept a wrong formatting (in the view of the robot) during review/approval because it still improves the overall code base.
I understand that we have a limitation here with the script format tool and how we integrate it into a github workflow. We can only check the whole file and not just the diff that a commit/PR introduces. This is why we have to live with it and understand the we cannot side track.